src/coord/src/coord.rs

```diff
@@ -3648,13 +3648,14 @@ where
                         |rows: Vec<Row>| -> Result<Vec<(Row, Diff)>, CoordError> {
                             // Use 2x row len incase there's some assignments.
                             let mut diffs = Vec::with_capacity(rows.len() * 2);
+                            let mut datum_vec = repr::DatumVec::new();
```

Comment:
is it possible and worth it to move this datum_vec up to where we create the arena on line 3644?

Reply:
I put it here on the theory that .. in other cases we don't need it, and afaict it isn't any different here vs there. I could be wrong about that, though! But, I didn't know how often we are in this one case vs the others, is why I did it this way.